examples/explorer/src/helpers/blockHelper.js

```diff
@@ -7,3 +10,25 @@ export const isBlockNumber = searchValue => {
 export const blocksFromBlockConnection = connection => {
   return connection.edges.map(edge => edge.node);
 };
+
+// This function will only return the chainLengths
+// not the full block objects
+export const getNextPrev = async block => {
+  const prev = havePrevious(block) && Number(block.chainLength) - 1;
+  const next = (await haveNext(block)) && Number(block.chainLength) + 1;
+
+  return { prev, next };
+};
+
+// FIXME: should have some logic to really know if
+// there are next epoch or not.
+export const haveNext = async block => {
+  const { status } = await fetchQuery(LastBlockQuery, {});
+  const lastBlock = status.latestBlock;
+
+  return !(lastBlock.chainLength === block.chainLength);
```

Comment:
I think it'd be a bit more clear to check that is greater? like: `lastBlock,chainLength > block.chainLength`. Although is not really important.